src/sonic_ax_impl/mibs/vendor/cisco/ciscoSwitchQosMIB.py

```diff
@@ -130,6 +135,46 @@ def update_stats(self):
                         self.mib_oid_list.append(mib_oid)
                         self.mib_oid_to_queue_map[mib_oid] = counter_value
 
+        # Update queue counters for LAG
+        lag_range = sorted(list(self.oid_lag_name_map.keys()))
+        for lag_index in lag_range:
+            lag_oid_list = []
+            lag_oid_to_queue_map = {}
+            # Get counters for each LAG member
+            for lag_member in self.lag_name_if_name_map[self.oid_lag_name_map[lag_index]]:            
+                lag_member_queues = []
+                try:
+                    lag_member_queues = self.port_queue_list_map[mibs.get_index(lag_member)]
+                except KeyError:
```

Comment:
>KeyError [](start = 23, length = 8)

The same.

Reply:
It doesn't mean the DB corruption.
It means that port does not have configured queue counters.
The appropriate comment is already added.
Which information do we need to extend the current comment?

Reply:
Refactored to no exceptions code